src/openapi/streaming/streaming.js

```diff
@@ -150,10 +150,16 @@ function onConnectionStateChanged(change) {
             break;
 
         case this.CONNECTION_STATE_RECONNECTING:
+            // logs when longPolling is enabled after trying webSockets
+            if (signalRTransport && signalRTransport !== this.currentTransport && signalRTransport.name === 'longPolling') {
+                log.warn(LOG_AREA, 'changing transport to long polling', true);

```

Comment:
Isn't 3rd argument used for additional data? And 4th one for enabling persisting of warn?

Reply:
It is...